test/commons/math/rects-touch.js

```diff
@@ -0,0 +1,50 @@
+describe('math.rects-touch', function () {
+  const rectsTouch = axe.commons.math.rectsTouch;
+
+  let rectA, rectB;
+  beforeEach(() => {
+    rectA = {
```

Comment:
Not strictly necessary, but you could consider using DOMRect in these tests. (I only learned of these a week ago, otherwise I might've used them in target-size stuff.)

Reply:
Not a bad idea. If the tests were more complicated I might consider that, but since it's a simple test I'll just leave it as is.